server/knexfile.js

```diff
@@ -43,5 +44,20 @@ module.exports = {
     seeds: {
       directory: './data/seeds',
     },
-  }
-};
\ No newline at end of file
+  },
+  production: {
+    client: 'pg',
+    connection: {
+      filename: process.env.DATABASE_URL,
+    },
+    useNullAsDefault: true,
```

Comment:
I don't think you need `useNullAsDefault`, the migration directory, or `seeds`. This was my whole production object from the backend project that worked fine:

```
  production: {
    client: "pg",
    connection: dbConnection,
    pool: {
      min: 2,
      max: 10
    },
    migrations: {
      tableName: "knex_migrations"
    }
  }
```

Reply:
That's correct--you only need useNullAsDefault for SQLite3 databases, not pg ones.